embc-app/ClientApp/src/app/self-registration/self-registration-two/self-registration-two.component.html

```diff
@@ -70,22 +70,14 @@ <h2 class="blue-header h3">INSURANCE</h2>
 
   <h2 class="blue-header h3">NEXT STEPS</h2>
   <section class="padded-section">
-    <p class="required">
-      While you are evacuated do you require supports?<br>
-      This could be in the form of food / lodging / clothing / incidentals or transportation.
+    <p>
+      While you are evacuated you may require emergency supports to ensure you and your family’s needs are met. An Emergency Support Services volunteer will meet with you in a Reception Centre in order to determine these needs.
     </p>
-    <div class="form-group">
-      <div class="form-check">
-        <input type="radio" [class.is-invalid]="invalid('requiresSupport')" class="form-check-input" [value]="true" formControlName="requiresSupport" id="requiresSupportTrue">
-        <label class="form-check-label" for="requiresSupportTrue">Yes</label>
-      </div>
-      <div class="form-check">
-        <input type="radio" [class.is-invalid]="invalid('requiresSupport')" class="form-check-input" [value]="false" formControlName="requiresSupport" id="requiresSupportFalse">
-        <label class="form-check-label" for="requiresSupportFalse">No</label>
-      </div>
-      <span [class.d-block]="invalid('requiresSupport')" class="invalid-feedback">{{validationErrors['requiresSupport']}}</span>
-    </div>
-    <div *ngIf="ui.showAvailableServices()" class="feature-block">
+    <p>
+      To expedite this process, please indicate <strong>which categories of support you are able to provide for yourselves.</strong>
+
```

Comment:
Please remove extra line break here. If you need extra space visually, add a margin to this element or the one after it. (Basic principle: without CSS this page should still be easily readable.)